src/index.html

```diff
@@ -11,6 +11,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    class="search-form"
+    method="post"
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    data-qa="big"
+    >
+      <label for="search-input">
+        <input
+            class="input"
+            type="search"
+            name ="search"
+            id="search-input"
+            data-qa="keypress"
+            placeholder="Try ''Los Angeles''"
+        >
+        <img
+        class="icon"
+        src="images/Search.svg"
+        alt=" "
+        >
```

Comment:
Go through [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) one more time and fix your attributes indents, it should be 2 spaces

Reply:
done